data/strings/types_strings.txt

```diff
@@ -17550,7 +17577,7 @@
     mr = परिसर
     nb = Sted
     nl = Plaats
-    pl = Lokalne
+    pl = Miejsce niezamieszkane
```

Comment:
```suggestion
    pl = Nazwane miejsce niezamieszkane
```

Reply:
that is not marking uninhabited areas, it is marking named spots that are not fitting other categories of named objects

Reply:
Yes, I agree, that's a better translation.

Reply:
In such case you should have button allowing to apply this suggestion to your PR.